src/svgs/CheckIcon.tsx

```diff
@@ -1,39 +1,16 @@
 import React from "react"
-import { color } from "../helpers"
-import { Color } from "../Theme"
 import { Icon, IconProps } from "./Icon"
 
-interface CheckIconProps extends IconProps {
-  fill?: Color
-  /** Apply additional styles to component */
-  style?: object
-  onClick?: (e: any) => void
+/** CheckIcon */
+export const CheckIcon: React.SFC<IconProps> = props => {
+  return (
+    <Icon {...props} viewBox="0 0 18 18">
+      <title>check</title>
+      <path
+        d="M6.936 12.206l7.64-7.63.848.849-8.492 8.48-4.248-4.282.852-.846z"
+        fill="#000"
```

Comment:
E.g., the green checks that we see around will now all be black because `fill` is no longer  present. 

if there's a way to hoist fill properties up to `Icon` would be ideal (every icon would then support color) but I'm not sure of the best path forward for this.

Reply:
This works now:

```
<CheckCircleIcon fill="green100" />
```

Gives you this:

<img width="140" alt="Screen Shot 2019-03-11 at 3 27 47 PM" src="https://user-images.githubusercontent.com/79799/54155515-48068e00-4412-11e9-9670-e28c3a1e6080.png">